src/lang/index.js

```diff
@@ -1,4 +1,75 @@
-// TODO read/write language setting to local cookie
-import en from './en.json';
+const savedLang = localStorage.getItem('localization');
+const lang = require(`./${savedLang || 'en'}.json`);
 
-export default en;
+export const langs = [
+  {
+    value: 'en',
+    en: 'English',
```

Comment:
Basically each language file should have the name of every language in that language, with an additional one called "native"